onyx.json

```diff
@@ -0,0 +1,28 @@
+{
+    "response": [
+        {
+            "maintainer": "Vikas(Vikas_2_0)",
+            "oem": "poco",
+            "device": "poco F7",
+            "filename": "crDroidAndroid-16.0-20251103-onyx-v12.2.zip",
+            "download": "https://sourceforge.net/projects/crdroid/files/onyx/12.x/crDroidAndroid-16.0-20251103-onyx-v12.2.zip/download",
+            "timestamp": 1762142911,
+            "md5": "d5a363bcedb11f815a319b78462863ef",
+            "sha256": "64d43865153ca13ecf6d99333f8f6d32467018f339a70273e21172b2c049401a",
+            "size": 3197249498,
+            "version": "12.2",
+            "buildtype": "Monthly",
+            "forum": "https://xdaforums.com/t/rom-poco-f7-onyx-16-0-crdroidandroid-v12-x-02-11-2025.4766256/",
+            "gapps": "https://sourceforge.net/projects/nikgapps/files/Elite-Releases/ionut/Android-16",
+            "firmware": "https://xmfirmwareupdater.com/hyperos/onyx/",
+            "modem": "",
+            "bootloader": "",
+            "recovery": "https://sourceforge.net/projects/crdroid/files/onyx/12.x/recovery/",
+            "paypal": "https://www.paypal.me/vikasyaduvanshii",
+            "telegram": "https://t.me/crdroidonyx",
+            "dt": "https://github.com/crdroidandroid/android_device_xiaomi_onyx",
+            "common-dt": "",
+            "kernel": "https://github.com/crdroidandroid/android_device_xiaomi_onyx-kernel"
```

Comment:
please correct to use standard naming scheme for repos
https://github.com/crdroidandroid/android_kernel_xiaomi_onyx